src/templates/oauth/mixins/user_cover_src_mixin.html

```diff
@@ -1 +1 @@
-{% load staticfiles %}{% if userprofile.cover %}{{ userprofile.cover.crop.850x315 }}{% elif userprofile.year == '1' %}{% static 'assets/others/cover2.svg' %}{% elif userprofile.year == '2' %}{% static 'assets/others/cover1.svg' %}{% elif userprofile.year == '3' %}{% static 'assets/others/cover4.svg' %}{% elif userprofile.year == '4' %}{% static 'assets/others/cover3.svg' %}{% elif userprofile.year == '5' %}{% static 'assets/others/cover3.svg' %}{% endif %}
\ No newline at end of file
+{% load staticfiles %}{% if userprofile.cover %}{{ userprofile.cover.crop.850x315 }}{% elif userprofile.year == '1' %}{% static 'assets/others/cover2.svg' %}{% elif userprofile.year == '2' %}{% static 'assets/others/cover1.svg' %}{% elif userprofile.year == '3' %}{% static 'assets/others/cover4.svg' %}{% elif userprofile.year == '4' %}{% static 'assets/others/cover3.svg' %}{% elif userprofile.year == '5' %}{% static 'assets/others/cover3.svg' %}{% else %}{% static 'assets/others/cover4.svg' %}{% endif %}
```

Comment:
i am not sure if this is correct

Reply:
![image](https://user-images.githubusercontent.com/32496777/42685848-ce24967c-86b1-11e8-99f1-0fb02c8fad95.png)
 varna aisa dikhega